--- a/browser/installer/package-manifest.in	2026-09-13 05:22:57
+++ b/browser/installer/package-manifest.in	2026-09-13 05:22:57
@@ -275,6 +275,7 @@
 @RESPATH@/toji.cfg
 ; Toji ships enterprise policies and built-in add-ons (not only Mozilla builds).
 @RESPATH@/distribution/*
+@RESPATH@/toji-agent-server
 @RESPATH@/browser/defaults/permissions
 ; Remote Settings JSON dumps
 @RESPATH@/browser/defaults/settings
